wordfreq_builder/wordfreq_builder/ninja.py

```diff
@@ -264,7 +292,10 @@ def subtlex_en_deps(dirname_in, languages):
         )
 
     output_file = wordlist_filename('subtlex-en', 'en', 'counts.txt')
-    add_dep(lines, 'merge_counts', processed_files, output_file)
+    add_dep(
+        lines, 'merge_counts', processed_files, output_file,
+        params={'cutoff': 0}
```

Comment:
Is this necessary? `read_values()` already uses 0 as a default value for `cutoff`.